src/main/java/de/zalando/aruha/nakadi/controller/EventPublishingController.java

```diff
@@ -26,6 +38,8 @@
 
     private final TopicRepository topicRepository;
     private final EventTypeRepository eventTypeRepository;
+    private final ValidationStrategy validationStrategy = new EventBodyMustRespectSchema();
+    private final ValidationStrategyConfiguration vsc = new ValidationStrategyConfiguration();
```

Comment:
Is it a temporary solution for now? Because I believe we will need some validation strategies registry or something like that when we have more than 1 strategy :)